src/sql/workbench/browser/modelComponents/media/table.css

```diff
@@ -3,21 +3,52 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-.center-align
-{
+.center-align {
 	text-align: center;
 }
 
-.align-with-header
-{
-	padding-left:3px !important;
+.align-with-header {
+	padding-left: 3px !important;
 }
 
-.no-borders
-{
+.no-borders {
 	border: none !important
 }
 
 .display-none {
 	display: none;
-}
\ No newline at end of file
+}
+
+.detailView-toggle {
```

Comment:
Scope these down so they're all under the table component element (modelview-table)